scripts/test.ps1

```diff
@@ -172,7 +177,10 @@ param (
     [switch]$UseQtip = $false,
 
     [Parameter(Mandatory = $false)]
-    [string]$OsRunner = ""
+    [string]$OsRunner = "",
+
+    [Parameter(Mandatory = $false)]
+    [int]$NumIterations = 0
```

Comment:
Instead of defaulting this to `0`, default to `1` and then always do the loop below, put only print out iteration to console if `NumIterations > 0` to simplify things a bit.

Reply:
Sounds good